packages/ethereum-contracts/contracts/agreements/gdav1/SuperfluidPool.sol

```diff
@@ -474,8 +454,19 @@ contract SuperfluidPool is ISuperfluidPool, BeaconProxiable {
             assert(GDA.appendIndexUpdateByPool(superToken, p, t));
         }
         emit MemberUnitsUpdated(superToken, memberAddr, oldUnits, newUnits);
+    }
 
-        _handlePoolMemberNFT(memberAddr, newUnits);
+    // replicates GDAv1._isPool in order to eliminate unnecessary gas cost (less external calls)
```

Comment:
It's a low hanging fruit of not violating DRY: Extract the storage related function as a global "free" function so that reusing is strictly inlining same code in two different contracts (gda logic or the pool logic).

Reply:
if you want faster 1.13 progress, removing this particular change is also okay for me. But I ask for no code quality degradation by not repeating code.